### PR TITLE
[miio] add support 5 zhimi heaters

### PR DESCRIPTION
**Pull request by marcelrv**
add new
zhimi.heater.ma2 Mi Smart Space Heater S
zhimi.heater.ma3 Mi Smart Baseboard Heater E
zhimi.heater.mc2 Mi Smart Space Heater S
zhimi.heater.na1 Smartmi Smart Fan
zhimi.heater.nb1 Smartmi Smart Fan Heater

improving existing
zhimi.heater.za1 Smartmi Radiant Heater Smart Version
zhimi.heater.za2 Smartmi Smart Convector Heater 1S
zhimi.heater.zb1 Smartmi Smart Convector Heater 1S

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

**Comment by cpmeister** [comment]
Please resolve conflicts.

**Comment by marcelrv** [comment]
thanks!  conflict is resolved.

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json [hunk 1]
```diff
@@ -170,20 +229,24 @@
 				"piid": 7,
 				"friendlyName": "Private-Service - Use Time",
 				"channel": "use-time",
-				"channelType": "time",
-				"type": "Number:Time",
+				"type": "Number:Duration",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json:232]: I don't think that Number:Duration is a valid type. Was this something recently added to core?

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json:232]: You are probaly right..
I did see it in few other bindings
https://github.com/openhab/openhab-addons/search?l=XML&q=number%3Aduration
https://github.com/openhab/openhab-addons/blame/main/bundles/org.openhab.binding.teleinfo/README.md#L83

I'll change it...

File: bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json [hunk 2]
```diff
@@ -85,6 +113,12 @@
 				"channelType": "time",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json:113]: Why did you remove `"channelType": "time",` from some of the devices but not others?

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json]: I nowadays remove the channeltype (unless very specific reasons) because than it is bit more transparant and less error prone.

In this case... it was just because I missed removing this one. :-(

**Review by cpmeister — APPROVED** [review]
